AutoMLTest/Program.cs

```diff
@@ -55,17 +58,23 @@ static int Main(string[] args)
 
             var dataSplit = mlContext.Data.TrainTestSplit(trainValidateData, testFraction: 0.2);
 
-            // Configure AutoML
-            var myTrialRunner = new MyRunner(mlContext, dataSplit.TrainSet, dataSplit.TestSet);
```

Comment:
There's no need to inject a custom trial runner for multi-classification automl experiment.